### DISCUSSION TITLE
Installation guide hint

### DISCUSSION THREAD

**Discussion by saschabrockel** [RandomNinjaAtk/arr-scripts#51] [Q&A] [answered]
Please add the hint that the init bash script [script_init.bash](https://github.com/RandomNinjaAtk/arr-scripts/blob/main/radarr/scripts_init.bash) must have the end of line sequence `LF `and not `CRLF`! Otherwise, it will crash and not work. It is a little hint but very useful for unexperienced users :)

Should be added for all folders.

**Comment by RandomNinjaAtk** [comment, marked as answer]
If your downloading/saving the file using the raw link, what your asking for is not needed...

The wiki is open for edits, feel free to add tutorials/instructions there... And I can post the link to the wiki to each readme...
